spec/controllers/roles_controller_spec.rb

```diff
@@ -19,7 +17,7 @@ def prime_minister
 
     it "when the content item exists" do
       get :show, params: { name: role }
-      assert_response :success
+      expect(response).to have_http_status(:success)
```

Comment:
huh, it found `assert_response` here. Maybe it's looking for the :symbol invocation. 🤔